docs/tutorial.rst

```diff
@@ -3,6 +3,96 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package that will
+be detailed in this tutorial. The first is via the command line interface (CLI). This is
+useful if you want to do a straightforward run of the models in the ``stdpopsim``
+catalog. The other way to access the ``stdpopsim`` resources is via the python API. This
+is a bit more complicated however it allows for more advanced tasks. This tutorial will
+walk through both ways of using the ``stdpopsim`` package as well as some more advanced
+tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+The CLI provides access to the catalog of models that have already been implemented by
```

Comment:
Provide a link to the catalog.

Reply:
Yup, there are a number of places in this tute where it would be great to link to the catalog directly. Btw, you can do this by replacing the word `catalog` with ```:ref:`Catalog <sec_catalog>` ```

Reply:
Thanks for the howto! I'll get on it.